.github/workflows/pythonpackage.yml

```diff
@@ -10,11 +10,46 @@ on:
     branches: [ master ]
 
 jobs:
+
   build:
```

Comment:
rename to `build-commons` to follow the same pattern?

Just in case we create more commons libraries in this repo.